<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(167, 205, 255);
	border:none;
    padding: 3px;
	font-family: &quot;微软雅黑&quot;;
	font-size: 20px;
	text-align: center;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
	background-color: rgb(85, 170, 255);
	text-align: right;
	padding-right: 20px;
	font-weight:100
}
QPushButton:hover
{
	background-image: url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/icons/dark/appbar.navigate.next.png&quot;);
	background-repeat:no-repeat;
	background-position: center left;
}
QPushButton:pressed, QPushButton:checked
{
	background-image: url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/icons/light/appbar.navigate.next.png&quot;);
	background-repeat:no-repeat;
	background-position: center left;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="filter_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>数据过滤</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feature_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>特征提取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>模型训练</string>
    </property>
   </widget>
   <widget class="QPushButton" name="params_plot_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>图形绘制</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
